feat: convert relative urls to absolute (#125)
peter-evans/dockerhub-description: README.md

| `repository` | Docker Hub repository in the format `<namespace>/<name>`. | `github.repository` |
| `short-description` | Docker Hub repository short description. | |
| `readme-filepath` | Path to the repository readme. | `./README.md` |

> | `enable-url-completion` | Enables completion of relative urls to absolute ones. See also [Known Issues](#known-issues). | `false` |

Reviewer: ```suggestion
| `enable-url-completion` | Enables completion of relative URLs to absolute ones. See also [Known Issues](#known-issues). | `false` |
```

> | `image-extensions` | Extensions of files that you be treated as images. | `bmp,gif,jpg,jpeg,png,svg,webp` |

Reviewer: ```suggestion
| `image-extensions` | File extensions that will be treated as images. | `bmp,gif,jpg,jpeg,png,svg,webp` |
```
